runtime/src/accounts_db.rs

```diff
@@ -753,6 +776,66 @@ impl AccountsDB {
         (reclaims, dead_keys)
     }
 
+    // Construct a vec of the keys needed for cleaning from uncleaned_keys and
+    // zero_lamport_keys sets.
+    fn get_clean_keys(
```

Comment:
nits: this code is the meat of this pr? so, `get_` sounds too humble. how about `fn construct_`/`fn create_`/`fn calculate_`.

Reply:
or `clean_candidate_pubkeys`?